app/main.py

```diff
@@ -1,3 +1,13 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    cache_lib = {}
```

Comment:
why the name of the variable contains `lib`?

Reply:
because it looks like a library.. That's why I named it in this way. Is it wrong?